app/models/concerns/bulkrax/import_behavior.rb

```diff
@@ -21,6 +21,8 @@ def build_for_importer
         status_info(e)
       else
         status_info
+      ensure
+        self.save!
```

Comment:
Occasionally, Entries were being processed without saving at the end. This would lead to imported records and "Complete" statuses without the entry having any parsed metadata set